### ISSUE TITLE
[Bug] "ReferenceError: Buffer is not defined" when datasetToBlob is called

### ISSUE THREAD

**Issue by dxlin** [OHIF/Viewers#3588]
### Describe the Bug

"ReferenceError: Buffer is not defined" when datasetToBlob is called (i.e. export RT report).

### Steps to Reproduce

1. Start viewer
2. Select Water Phantom study (or any with CT and PT) in TMTV mode
3. Create "New Label" on right right
4. Select "Rectangle ROI Threshold Tool"
5. Create arbitrary rectangle in axial PT viewport
6. Click "Run"
7. Click "Create RT Report"

### The current behavior

Due to "ReferenceError: Buffer is not defined" when datasetToBlob is called, application fails to create a file to be saved to local directory.

### The expected behavior

File blob should be create from datasetToBlob, which can then be used to download file.

### OS

Windows 10

### Node version

18.16.0

### Browser

Edge v115.0.1901.188

**Comment by sedghi** [comment]
yeah buffer is in node,  you should add the Buffer as dependency, see the @cornerstonejs/adapters

**Comment by dxlin** [comment]
I've tried adding "buffer" as a dependency to package.json as well as to:
extensions\default\package.json
extensions\tmtv\package.json
extensions\cornerstone-dicom-sr\package.json

But ended up with the same error. I did find a possible solution and made a PR:
https://github.com/OHIF/Viewers/pull/3590
